datahub-upgrade/build.gradle

```diff
@@ -23,6 +25,9 @@ dependencies {
     exclude group: "org.apache.htrace", module: "htrace-core4"
     exclude group: "org.eclipse.jetty", module: "jetty-util"
     exclude group: "org.apache.hadoop.thirdparty", module: "hadoop-shaded-protobuf_3_7"
+    exclude group: "com.charleskorn.kaml",module:"kaml"
```

Comment:
Couple of linting issues here, please add a space after the `,` to match the others. Thanks!